### DISCUSSION TITLE
code review IM-13

### DISCUSSION THREAD

**Discussion by Valentyn13** [HowProgrammingWorks/LiveQA#83] [Code review]
Первєєв Валентин, група ІМ-13.
Написав звичайний todolist
Посилання на відкритий репозиторій:[https://github.com/Valentyn13/Todo-App](https://github.com/Valentyn13/Todo-App)

**Comment by tshemsedinov** [comment]
See review: https://github.com/Valentyn13/Todo-App/issues/2

**Reply by Valentyn13** [reply]
Виправив всі ваші зауваження.
Посилання [на нову вітку](https://github.com/Valentyn13/Todo-App/tree/fixed-issue)
